.circleci/config.yml

```diff
@@ -180,16 +172,16 @@ jobs:
           name: Build Zalpha
           command: |
             if [[ ${APP_VERSION} == *"-rc-"* ]]; then
-              bundle exec fastlane build_alpha skip_confirm:true skip_prechecks:true create_release:true upload_to_play_store:true
+              bundle exec fastlane build_alpha skip_confirm:true skip_prechecks:true create_release:false upload_to_play_store:false
```

Comment:
Introduced via temporary commit 3bb2b8c4a797d6cdc1a01863f6b4dc9e236fa5e2, which I made in order to be able to test the PR by trigger a Release Build… without creating a GitHub Release nor a PlayStore entry.

🚨 So just a reminder that this commit needs to be reverted once we've tested everything (and ensured `fastlane trigger_release_build` succeeds) before we finally merge this PR.